source/postprocess/visualization/adiabat.cc

```diff
@@ -33,9 +33,12 @@ namespace aspect
       Adiabat<dim>::
       Adiabat ()
         :
-        DataPostprocessor<dim> ()
+        DataPostprocessor<dim> (),
+        Interface<dim>("K,kg/s/s/m,kg/m/m/m,kg/m/m/m/m")
```

Comment:
I appreciate that you stick very close to the basic SI units, but could you still use `Pa` for units of pressure and stress and `Pa*s` for viscosity? Not many users will recognize `kg/s/s/m` as `Pa`